src/styles/style.scss

```diff
@@ -434,6 +462,44 @@ label {
   }
 }
 
+#buttons-container2D {
+  display: flex;
+  justify-content: space-between;
+}
+
+.popover-wrapper {
```

Comment:
Will be perfect to put it inside `popover.scss` file as standard style for popovers.

Reply:
Please check comment for line: 368, to get correct path and file structure for styles.